src/recipes/recipes.jl

```diff
@@ -5,7 +5,7 @@
       x --> longitudes(layer)
       y --> latitudes(layer)
       z --> layer.grid
-   else if get(plotattributes, :seriestype, :histogram) in [:histogram, :density]
+   elseif get(plotattributes, :seriestype, :histogram) in [:histogram, :density]
```

Comment:
Nice. While you're at it, it is somewhat better to use `in (:histogram, :density)` as the tuple will stack allocate, while the currently used Array will heap allocate. Should we fix this while we're at it?

Reply:
@tpoisot do you plan to activate CI for this repo?

Reply:
Very much so, especially since @gabrieldansereau is now using it for actual research